recipes-support/os-helpers/os-helpers/os-helpers-sb

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# Copyright 2024 Balena Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# vi: ft=sh
+
+# shellcheck disable=SC1091
+[ -f "/usr/libexec/os-helpers-logging" ] && . /usr/libexec/os-helpers-logging
+[ -f "/usr/libexec/os-helpers-fs" ] && . /usr/libexec/os-helpers-fs
+
+BALENA_NONENCRYPTED_BOOT_PARTITION_FILES="\
+	@@BALENA_IMAGE_FLAG_FILE@@ \
+	imx-boot-${MACHINE}-sd.bin-flash_evk \
+	ucm-imx8m-mini.dtb \
+	sb-iotgimx8.dtb \
+	sb-iotgimx8-can.dtb \
+	sb-iotgimx8-ied.dtb \
+	Image.gz \
+	bootenv \
+"
+
+is_secured() {
+	if @@IS_SECURED_CHECK_SKIP@@; then
+		warn "ASSUMING SECURED - DO NOT USE IN PRODUCTION"
```

Comment:
I would word this differently, "ASSUMING SECURED" is not scary enough :). "SKIPPING SECURE BOOT CHECK" or "IGNORING SECURE BOOT CHECK" sounds better.

Reply:
I can change it - but it is also an OS_DEVELOPMENT only feature so I wasn't trying so scare the developer. Using an OS_DEVELOPMENT build would be just as insecure as setting this.
```
BALENA_IS_SECURED_CHECK_SKIP = "${@bb.utils.contains('DISTRO_FEATURES', 'osdev-image', "${IS_SECURED_CHECK_SKIP}", '0', d)}"
```

Reply:
Ah I see, if it is a build-time config, then it's fine.